plexpy/webserve.py

```diff
@@ -1180,13 +1183,28 @@ def getLog(self, start=0, length=100, **kwargs):
             search_value = kwargs.get('search[value]', "")
 
         if 'search[regex]' in kwargs:
-            search_regex = kwargs.get('search[regex]', "")
+            search_regex = kwargs.get('search[regex]', "") # should this be removed?
+
+        filt = []
+        fa = filt.append
+        with open(os.path.join(plexpy.CONFIG.LOG_DIR, 'plexpy.log')) as f:
+            for l in reversed(f.readlines()):
+                try:
+                    temp_loglevel_and_time = l.split('- ')
```

Comment:
I think this leaves a trailing space on the log time. `split(' - ')` instead?

Again being picky.